### PR TITLE
Bug/default args constant rigid body properties

### PR DESCRIPTION
**Pull request by soufyanR1**

### CODE REVIEW

File: tudatpy/kernel/expose_numerical_simulation/expose_environment_setup/expose_rigid_body_setup.cpp [hunk 1]
```diff
@@ -44,9 +44,9 @@ void expose_rigid_body_setup(py::module &m) {
 //
     py::class_<tss::RigidBodyPropertiesSettings, std::shared_ptr<tss::RigidBodyPropertiesSettings>>(
         m, "RigidBodyPropertiesSettings",
-        get_docstring("RigidBodyPropertiesSettings").c_str())
```
Comment (DominicDirkx) [inline tudatpy/kernel/expose_numerical_simulation/expose_environment_setup/expose_rigid_body_setup.cpp:47]: Why was this removed? Did it cause a compilation error or a documentation error?

Comment (soufyanR1) [reply tudatpy/kernel/expose_numerical_simulation/expose_environment_setup/expose_rigid_body_setup.cpp:47]: I interpreted the google doc wrongly on this https://github.com/tudat-team/tudatpy/pull/136/commits/2992fcdbf9fb137e25d58ae7865dbcf128a5189f commit. This should not actually be removed, my bad
